src/Domain/Infra/Doctrine/MappingCacheWarmer.php

```diff
@@ -34,7 +36,9 @@ public function warmUp($cacheDir): void
         $filesystem->mkdir($target = $cacheDir.'/'.$this->dirName);
 
         foreach ($this->mappingFiles as $file) {
-            $filesystem->copy($file, $target.'/'.basename($file));
+            $contents = file_get_contents($file);
+            $contents = str_replace('%msgphp.doctrine.key_max_length%', $this->keyMaxLength, $contents);
```

Comment:
let's move this to `Doctrine\MappingConfig` e.g. `MappingConfig::interpolate(string $contents)` which replaces its own property placeholders.

Reply:
I've created such a class in commit: https://github.com/msgphp/msgphp/pull/214/commits/ae59c6c34d28d4059cf9b738b8f4d4d6daaad338. Was this what you meant?

As a side effect I had to enable the autowireing for the make command as well as the `CacheWarmer`. IMO this should be fine. Any objection against it?